.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -44,7 +44,7 @@ body:
     attributes:
       label: What type of installation are you running?
       description: >
-        If you don't know, you can find it in: Configuration panel -> Info.
+        Can be found in: Configuration panel -> Info.
```

Comment:
Bonus: This accepts markdown, so we can add a My home assistant link/button here.

https://my.home-assistant.io

Reply:
That's neat!

I must've disabled "my" because I don't have it installed, so I can't easily test.

If you think using it here is best, I'm open to your suggestion on how best to link it! 💜